canonical/charmhub.io
# chore: update dependencies

## Done
- Updates dependency issues (mainly Vanilla and store-component conflicts)
- There are now some Sass deprecation warnings related to the new Vanilla version, but for now, no content is affected

## How to QA
- All tests pass
- Click around the demo and make sure everything looks as expected

## Testing
